__tests__/ga.test.js

```diff
@@ -44,17 +46,15 @@ inputData.forEach((input, index) => {
 });
 
 describe(`${name} Tests`, () => {
-describe("Router Tests", () => {
-  it("Payload", async () => {
-    let singleRouterOutput, singleExpectedOutput;
-    const routerOutput = await transformer.processRouterDest(inputRouterData);
-    routerOutput.forEach((output, index) => {
-      singleRouterOutput = routerOutput[index];
-      singleExpectedOutput = expectedRouterData[index];
-      singleExpectedOutput.batchedRequest.params.qt = singleRouterOutput.batchedRequest.params.qt;
-    })
-    expect(routerOutput).toEqual(expectedRouterData);
+  describe("Router Tests", () => {
```

Comment:
why formatting got changed?

Reply:
It was not correct earlier if you had noticed a function with a function is in the same indent.